src/snowflake/cli/api/commands/flags.py

```diff
@@ -55,6 +67,8 @@ def callback(value):
     callback=_callback(lambda: cli_context_manager.connection_context.set_account),
     show_default=False,
     rich_help_panel=_CONNECTION_SECTION,
+    envvar=_snowflake_env_variable("ACCOUNT"),
+    show_envvar=False,
```

Comment:
I noticed that some params for connection options repeats. 
Maybe we could introduce a base for this option
```python 
_ConnectionOptionBase = typer.Option(
    show_default=False,
    rich_help_panel=_CONNECTION_SECTION, 
    show_env_var=False,
) 

AccountOption = _ConnectionOptionBase(
    callback=_callback(lambda: cli_context_manager.connection_context.set_account), 
    envvar=_snowflake_env_variable("ACCOUNT"),
)
```